webpack.config.js

```diff
@@ -29,6 +29,7 @@ const webpackConfig = {
     },
     resolve: {
         alias: {
+            "@": path.join(__dirname, "src"),
```

Comment:
Let me know if you need help with anything related to webpack 💙

Reply:
@evenstensberg *Actually*...we do :) - https://github.com/nasa/openmct/issues/2507#issuecomment-558763066. Any thoughts?